spec/bundler/plugin_spec.rb

```diff
@@ -33,6 +33,18 @@
     allow(index).to receive(:register_plugin)
   end
 
+  describe "list command" do
+    let(:opts) { { "version" => "~> 1.0", "source" => "foo" } }
+    before do
+      allow(index).to receive(:installed_plugins) { %W(plug1 plug2) }
+      allow(index).to receive(:plugin_commands).with("plug1") { %W(c1 c2) }
+      allow(index).to receive(:plugin_commands).with("plug2") { %W(c1 c2) }
```

Comment:
I don't feel like I'm really testing anything here with index being a double :-(
Is that is how it is supposed to be?